go.mod

```diff
@@ -14,5 +14,5 @@ require (
 	k8s.io/api v0.19.0
 	k8s.io/apimachinery v0.19.0
 	k8s.io/client-go v0.19.0
-	sigs.k8s.io/controller-runtime v0.6.2
+	sigs.k8s.io/controller-runtime v0.6.3
```

Comment:
There seems to be a lot of vendoring changes being introduced in this commit as a result of this change. Do you know why this is needed?

Reply:
that version is required in https://github.com/RHsyseng/operator-utils/blob/main/go.mod